index.js

```diff
@@ -1,116 +1,119 @@
-const faker = require('faker');
-const properCase = require('proper-case');
+var faker = require('faker');
+var properCase = require('proper-case');
 
 // List of all types of Fakers. We specify this explicitly since there is
 // no easy way to filter out these from the other objects on the faker module.
-const fakerTypes = [
-    "address",
-    "commerce",
-    "company",
-    "database",
-    "date",
-    "finance",
-    "hacker",
-    "image",
-    "internet",
-    "lorem",
-    "name",
-    "phone",
-    "random",
-    "system"
-]
+var fakerTypes = [
+  'address',
+  'commerce',
+  'company',
+  'database',
+  'datatype',
+  'date',
+  'finance',
+  'git',
+  'hacker',
+  'helpers',
+  'image',
+  'internet',
+  'lorem',
+  'mersenne',
+  'music',
+  'name',
+  'phone',
+  'system',
+  'time',
+  'vehicle',
+];
 
 // Creates the scaffolds for Enum options that Insomnia's Template Tags expect
-populateFakerOptions = function(someArray) {
+function populateFakerOptions(someArray) {
   return someArray.sort().map(function(key) {
-      return {
-          displayName: properCase(key),
-          value: key
-      }
+    return {
+      displayName: properCase(key),
+      value: key,
+    };
   });
 }
 // Creates Faker Sub Types and hides them so that they only show when the
 // parent Type is selected
-populateFakerSubOptions = function() {
-    return fakerTypes.map( function(fakerType) {
-        var fakerTypeOptions = populateFakerOptions(Object.keys(faker[fakerType]))
-        return {
-            displayName: properCase(fakerType),
-            type: 'enum',
-            defaultValue: "",
-            options: fakerTypeOptions,
-            hide: args => fakerType != args[0].value
-        };
-    });
+function populateFakerSubOptions() {
+  return fakerTypes.map(function(fakerType) {
+    var fakerTypeOptions = populateFakerOptions(Object.keys(faker[fakerType]));
+
+    return {
+      displayName: properCase(fakerType),
+      type: 'enum',
+      defaultValue: '',
+      options: fakerTypeOptions,
+      hide: function(args) { return fakerType !== args[0].value; },
+    };
+  });
 }
 
-populateFakerLocalizationOptions = function() {
-    return Object.keys(faker.locales).sort().map(function(locale) {
-        return {
-            displayName: locale,
-            value: locale
-        }
-    });
+function populateFakerLocalizationOptions() {
+  return Object.keys(faker.locales).sort().map(function(locale) {
+    return {
+      displayName: locale,
+      value: locale,
+    };
+  });
 }
 
 // Actual Template Tags export that Insomnia expects
 module.exports.templateTags = [{
-    name: 'faker',
```

Comment:
If you're going to run a syntax prettier on this code, please open up a PR with support for prettier or whatever so we can easily see the differences in this specific PR